Fix torrentPath migration

dcposch opened the pull request:
Fixes #448 

**We should test this on Windows before merging.**

**Test plan:**
1. Delete config.json
2. Install the current released version
4. Run, add a torrent, then quit
5. Run from this branch. Both default torrents and the new one you added should still work
6. Repeat 1-5 on Windows and on posix (Mac or Linux)

feross commented:
Nice! Thanks for figuring out what cases we needed to handle. Was on a plane when you sent this PR. Will check it out and test now.

dcposch commented:
Tested on 64-bit Windows 7. Installed Webtorrent 0.1.0, added a torrent, installed 0.3.3, added another torrent, then ran from this branch. The config file was migrated correctly.

feross commented:
LGTM
